### ISSUE TITLE
4.0.7 Deliver Function

### ISSUE THREAD

**Issue by comradeflats** [PikaMug/Quests#1785]
**Quests 4.0.7**

**I set up a Quest to deliver 3 cod to an NPC, once complete I would get the reward/quest points. I recreated this with different items and different rewards. Everytime I got the same response which was "Cod X3 is not a required item for this quest! The difference is 'display name or lore'**

I recreated this in 4.0.6 and it worked, so I'm guessing it's something to do with 4.0.7. I'm running Paper MC #259 server download. Hope to see it resolved! : )

**Comment by comradeflats** [comment]
![Screen Shot 2021-09-12 at 7 56 00 PM](https://user-images.githubusercontent.com/81211706/133002243-ba9a79b6-9803-45a8-87f0-29a874d59cce.png)

**Comment by Jrud** [comment]
https://user-images.githubusercontent.com/11812997/133145587-96f43f32-fc03-495f-97c6-0573b5354b92.mp4

Same.  Recorded a video of my setup.

Quests 4.0.7 
Citizens 2.0.28-b2369 
Paper 1.17.1-259

**Comment by PikaMug** [comment]
@Jrud Thanks for the report. Please give the following build a try:
https://ci.codemc.io/job/PikaMug/job/Quests/263/

**Comment by Jrud** [comment]
I updated my 7.0.6 JAR to that one, and the console says it loaded all of the quests and actions I had set up so I went ahead and tried my simple wheat quest.  First, the action didn't activate when I started it to give me the wheat and I got an error in the console with a stack trace saying it couldn't call the sendMessage function because this.locleManager was null.  The stack trace shows it was being called from the showObjectives function.  I assume the message probably happens just before the action command in the execution order and it errored out first, so that would be that problem.

So I manually gave myself a wheat, and then I tried to rightclick the shopkeep from my video to hand it to him, and it consumed the wheat and then I got another error on the sendMessage function saying the getLocaleManager() function returned a null.  This happened in the finishObjective function.

I remember from the latest patch notes you were doing something with the languages recently correct?  am I maybe missing some files that would normally have been installed if I wiped out my config folder for the plugin?
